modules/packages/Crypto.chpl

```diff
@@ -72,6 +72,14 @@ module Crypto {
    return localKeyPair;
   }
 
+  /* The `CryptoBuffer` class is a wrapper around the internal representation
```

Comment:
Thanks for adding these!

Once comment that applies to all of the doc-strings: If you intended for the `` `back tics` `` to result in `inline code like this`, then they'll need to be ``` ``double bac tics like this`` ``` (common restructured text gotcha)

Reply:
ahh sorry. I almost over look this everytime. Markdown habit.

Reply:
Btw it is not inline code. It just represents some type/variable. Does that still need double backticks?